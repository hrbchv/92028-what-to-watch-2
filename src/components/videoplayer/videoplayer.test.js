import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./videoplayer.jsx";

const imgSrc = `img/bohemian-rhapsody.jpg`;
const videoSrc = `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`;
const index = 3;
const isPlay = true;

describe(`FilmsList`, () => {
  it(`should renders correctly`, () => {

    const tree = renderer
      .create(<VideoPlayer
        imgSrc={imgSrc}
        videoSrc={videoSrc}
        index={index}
        isPlay={isPlay}/>
      ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
