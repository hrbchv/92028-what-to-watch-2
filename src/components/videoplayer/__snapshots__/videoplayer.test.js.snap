// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmsList should renders correctly 1`] = `
<video
  muted={true}
  poster="img/bohemian-rhapsody.jpg"
  preload="true"
/>
`;
